### ISSUE TITLE
Error when using euro € as currency

### ISSUE THREAD

**Issue by dneez** [gtg922r/obsidian-numerals#1]
Hi.
Thanks for the plugin.
Using € symbol for currency produces following error, when used:
> SyntaxError:Syntax error in part "€" (char 16)

Example math block:
```
```math
item1 = 1000.00$
item2 = 1000.00€
_```
```

**Comment by gtg922r** [comment]
Thanks for the feedback. I'll add additional currency symbols in the next pre-release

**Comment by gtg922r** [comment]
Fixed and available in [0.0.9: Additional currency support](https://github.com/gtg922r/obsidian-numerals/releases/tag/0.0.9)

```
item1 = 1000.00$
item2 = 1000.00€
```

Now renders as 
```
item1 = 1000.00$    → 1,000 $
item2 = 1000.00€    → 1,000 €
```

**Comment by dneez** [comment]
Thanks!
